_shared/project/README.md

```diff
@@ -0,0 +1,263 @@
+{% if cookiecutter.get("visibility") == "public" -%}
+<a href="{{ cookiecutter.__github_url }}/actions/workflows/ci.yml?query=branch%3Amain"><img src="https://img.shields.io/github/workflow/status/{{ cookiecutter.github_owner }}/{{ cookiecutter.slug }}/CI/main"></a>
+{% endif -%}
+{% if cookiecutter._directory == 'pypackage' -%}
+<a href="{{ cookiecutter.__pypi_url }}"><img src="https://img.shields.io/pypi/v/{{ cookiecutter.slug }}"></a>
+{% endif -%}
```

Comment:
```suggestion
{% if cookiecutter.get("visibility") == "public" -%}
    <a href="{{ cookiecutter.__github_url }}/actions/workflows/ci.yml?query=branch%3Amain"><img src="https://img.shields.io/github/workflow/status/{{ cookiecutter.github_owner }}/{{ cookiecutter.slug }}/CI/main"></a>
{% endif -%}
{% if cookiecutter._directory == 'pypackage' -%}
    <a href="{{ cookiecutter.__pypi_url }}"><img src="https://img.shields.io/pypi/v/{{ cookiecutter.slug }}"></a>
{% endif -%}
```

It might result in slightly less eye murder if we can indent blocks where we can. I think we're stripping whitespace here anyway, so it shouldn't matter?

Reply:
Sure. I've done this for the one-liners but I don't think this is possible for multi-line blocks: whitespace control only dedents the first line not the rest. IIRC this is a limitation of Jinja2: there's no way to indent a multiline block in the template without it getting indented in the output file